### ISSUE TITLE
Missing % symbol or just OCaml specific section

### ISSUE THREAD

**Issue by psb** [melange-re/melange-re.github.io#59]
Should [line 858](https://github.com/psb/melange-re.github.io/blob/master/docs/communicate-with-javascript.md?plain=1#L858) in `communicate-with-javascript.md` have two % signs for the Reason syntax as the text describes, or is this section (lines 847 to 860 specific to OCaml syntax and should be hidden when the reader selects the Reason syntax?

**Comment by jchavarri** [comment]
Good catch. The Reason snippet is using the right syntax, but we should adapt the text in lines 847-860 to that syntax as well. I will push a fix.

**Comment by jchavarri** [comment]
We are working to have some kind of online playground so it becomes easier to visualize how the same code looks in both syntaxes.
